frontend/src/components/ui/TableAnomalyDetection.tsx

```
@@ -0,0 +1,306 @@
+import { useMemo, useState } from 'react'
+
+import { useMetrics, useScanTableAnomalies, useTableAnomalies } from '@/features/metrics/hooks'
+import { type TableAnomalyDetection } from '@/features/metrics/types'
+import { formatUTCToKST } from '@/lib/utils'
+
+interface TableAnomalyDetectionProps {
+  className?: string
+}
+
+export function TableAnomalyDetection({ className = '' }: TableAnomalyDetectionProps) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'TableAnomalyDetection'. Consider to delete it or rename it.

'TableAnomalyDetection' is defined here:


(lint/suspicious/noRedeclare)